### PR TITLE
refactor: use datastore for owner_quota 

### PR DESCRIPTION
**Pull request by crowlKats**

### CODE REVIEW

**Review by ry — COMMENTED** [review]
So in this PR, Mongo is still being used and Datastore is only partially being used? What would be left to fully migrate to Datastore?

**Comment by crowlKats** [comment]
The other two tables. will be doing separate PR for each table as it makes it a lot easier not to break things (and review). I have partially ready stashes for those.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ jobs:
     env:
       AWS_DEFAULT_REGION: us-east-1
       AWS_BACKUP_REGION: eu-central-1
+      GOOGLE_PROJECT_ID: test-project
```
Comment (bartlomieju) [inline .github/workflows/ci.yml:15]: Is this a correct value? Seems like a placeholder

**Review by bartlomieju — COMMENTED** [review]
Just one comment, otherwise it LGTM

File: .github/workflows/ci.yml (hunk 1 above)
Comment (crowlKats) [reply .github/workflows/ci.yml:15]: the value could be anything afaik

**Pull request merged**